project_event/tests/test_project_event_task.py

```diff
@@ -168,3 +169,26 @@ def test_070_cancel_action(self):
         self.assertEqual(
             calendar_event.state,
             'cancelled')
+
```

Comment:
Check all the previous test, there are some tests that did not pass

Reply:
Yes, it fails some tests
2018-12-18 21:40:08,599 155 **ERROR** 11.0_dec_18_07 odoo.addons.project_event.tests.test_project_event: FAIL

Reply:
@rimbendhaou @eilst  Changed and re-tested it is running fine